src/commands/suspend.js

```diff
@@ -0,0 +1,90 @@
+const Discord = require('discord.js');
+const getConfig = require('../config/get-config.js');
+const config = getConfig();
+module.exports = {
+  prefix: 'suspend',
+  description: 'Suspends a user for the given reason',
+  command: async function suspend(message) {
+    //check for appropriate permissions
+    if (message.member.hasPermission('KICK_MEMBERS') == false) {
+      message.channel.send(
+        `ERROR 401: ${message.author}, missing permissions.`
+      );
+      return;
+    }
+    const mod = message.author;
+    const msgArguments = message.content.split(' ');
+    const user = message.mentions.members.first();
+    //check for valid user tag
+    if (user == undefined) {
+      message.channel.send(`ERROR 400: ${mod}, invalid user tag.`);
+      return;
+    }
+    //cannot target self
+    if (user === mod) {
+      message.channel.send(`ERROR 400: ${mod}, cannot target self.`);
+      return;
+    }
+    const reasonArg = msgArguments.slice(3, msgArguments.length);
+    let reason = reasonArg.join(' ');
```

Comment:
Change to just 
```
const reason = reasonArg.join(' ') || 'ERROR 404: No reason provided.'
```

If this is an error shouldn't we exit?

Reply:
The command should still execute even if the user fails to provide a reason, right?
The "ERROR 404" is filler text for the embed.

Reply:
I'll reword this to remove the "ERROR" language, though.